src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -701,14 +797,20 @@ public Task<User> GetMeAsync(CancellationToken cancellationToken = default)
             bool disableNotification = default,
             int replyToMessageId = default,
             IReplyMarkup replyMarkup = default,
-            CancellationToken cancellationToken = default
+            CancellationToken cancellationToken = default,
+            bool allowSendingWithoutReply = default,
+            int heading = default,
+            int proximityAlertRadius = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.